crowbar_framework/app/models/backup.rb

```diff
@@ -110,7 +110,14 @@ def upgrade
   end
 
   def cleanup
-    @data.rmtree if @data
+    return unless @data || @data.to_s =~ /\A#{Rails.root.join("tmp").to_s}/
```

Comment:
My concern here is that you're relying on internal knowledge of how we patched `mktmpdir`. Can we somehow add a helper is the same location (`config/initializers/tmpdir.rb`) that gives us the parent directory so we don't rely on this internal knowledge (and don't break in case it changes)?